closeLoop0/publishHz: 20
closeLoop0/name: left_motor
closeLoop0/LPFAlpha: 0.7
closeLoop0/LPFHz: 50
closeLoop0/driverAddress: 128
closeLoop0/channel: 1
closeLoop0/port: 1
closeLoop0/PIDHz: 1000
closeLoop0/kP: 3.0
closeLoop0/kI: 3.0
closeLoop0/kD: 0
closeLoop0/limit: 1.0
closeLoop0/maxSpeed: 16.0
closeLoop0/cpr: 4288
closeLoop0/timeout: 1000
closeLoop0/motorType: 1
closeLoop0/direction: -1
closeLoop0/directionE: -1
closeLoop0/encoderType: 2
closeLoop0/port2: 3
closeLoop1/publishHz: 20
closeLoop1/name: right_motor
closeLoop1/LPFAlpha: 0.7
closeLoop1/LPFHz: 50
closeLoop1/driverAddress: 128
closeLoop1/channel: 2
closeLoop1/port: 2
closeLoop1/PIDHz: 1000
closeLoop1/kP: 3.0
closeLoop1/kI: 3.0
closeLoop1/kD: 0
closeLoop1/limit: 1.0
closeLoop1/maxSpeed: 16.0
closeLoop1/cpr: 4288
closeLoop1/timeout: 1000
closeLoop1/motorType: 1
closeLoop1/direction: -1
closeLoop1/directionE: 1
closeLoop1/encoderType: 2
closeLoop1/port2: 4
Diff/publishHz: 20
Diff/name: diff_driver
Diff/rWheel: 0.065
Diff/width: 0.255
Diff/baseLink: base_link
Diff/odom: odom_link
Diff/slip: 0.75
Diff/maxAng: 16.0
Diff/maxLin: 16.0
Diff/motorL: 0
Diff/motorR: 1
Battery/name: battery_monitor
Battery/pubHz: 3
Battery/voltageDividerRatio: 5.92
IMU/publishHz: 20
IMU/name: IMU
IMU/frameId: base_link
IMU/camp: 0
IMU_INIT: 1
GPS_INIT: 0
PPM_INIT: 0
BAT_INIT: 1
DIFF_INIT: 1
DIFF_INIT_OP: 0
closeLoopNum: 2
switchNum: 0
servoNum: 0
relayNum: 0
URFNum: 0
openLoopNum: 0
